packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -276,8 +281,15 @@ contract LongShortPair is Testable, Lockable {
     function _requestOraclePriceExpiration() internal {
         OptimisticOracleInterface optimisticOracle = _getOptimisticOracle();
 
-        // For now, we add no fees the the OO and set the reward to 0.
-        optimisticOracle.requestPrice(priceIdentifier, expirationTimestamp, customAncillaryData, collateralToken, 0);
+        // Use the prepaidProposerReward as the proposer reward.
+        collateralToken.safeApprove(address(optimisticOracle), prepaidProposerReward);
```

Comment:
Is there any potential savings in checking `prepaidProposerReward > 0` and if not, then no need to call `safeApprove`?

Reply:
Good point! Fixed.